Cargo.toml

```diff
@@ -24,3 +24,4 @@ version = "0.1.0"
 
 [workspace.dependencies]
 clap = "4"
+alloy-primitives = "0.8.15"
```

Comment:
please add a new line to remove this 
![image](https://github.com/user-attachments/assets/76aef8b5-6385-4a5a-b094-9d4fe1f732bc)